### PR TITLE
advanced, users: Document syncing extended attributes

### PR DESCRIPTION
**Pull request by calmh**
Ref https://github.com/syncthing/syncthing/pull/8513

### CODE REVIEW

File: users/config.rst [hunk 1]
```diff
@@ -114,6 +114,8 @@ The following shows an example of a default configuration file (IDs will differ)
             <junctionsAsDirs>false</junctionsAsDirs>
             <syncOwnership>false</syncOwnership>
             <scanOwnership>false</scanOwnership>
+            <syncXattrs>false</syncXattrs>
+            <scanXattrs>false</scanXattrs>
```
Comment (tomasz1986) [inline users/config.rst:118]:
```suggestion
            <sendXattrs>false</sendXattrs>
```

File: users/config.rst [hunk 2]
```diff
@@ -219,6 +221,8 @@ The following shows an example of a default configuration file (IDs will differ)
                 <junctionsAsDirs>false</junctionsAsDirs>
                 <syncOwnership>false</syncOwnership>
                 <scanOwnership>false</scanOwnership>
+                <syncXattrs>false</syncXattrs>
+                <scanXattrs>false</scanXattrs>
```
Comment (tomasz1986) [inline users/config.rst:225]:
```suggestion
                <sendXattrs>false</sendXattrs>
```

File: users/config.rst [hunk 3]
```diff
@@ -305,6 +309,8 @@ Folder Element
         <junctionsAsDirs>false</junctionsAsDirs>
         <syncOwnership>false</syncOwnership>
         <scanOwnership>false</scanOwnership>
+        <syncXattrs>false</syncXattrs>
+        <scanXattrs>false</scanXattrs>
```
Comment (tomasz1986) [inline users/config.rst:313]:
```suggestion
        <sendXattrs>false</sendXattrs>
```

Comment (tomasz1986) [reply users/config.rst:313]: By the way, there's still `scanOwnership` right above this 😉.

File: advanced/folder-send-xattrs.rst [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+==========
+
+.. versionadded:: 1.22.0
+
+When enabled, Syncthing will record extended attributes for local files when
+they are scanned and send this information to peer devices. Peer devices
+configured to :doc:`sync extended attributes <folder-sync-xattrs>` will use
+this information.
+
+Scanning extended attributes maybe have a performance impact on scan times.
```
Comment (acolomb) [inline advanced/folder-send-xattrs.rst:11]:
```suggestion
Scanning extended attributes may have a performance impact on scan times.
```

File: advanced/folder-sync-xattrs.rst [hunk 5]
```diff
@@ -0,0 +1,38 @@
+syncXattrs
+==========
+
+.. versionadded:: 1.22.0
+
+When enabled, Syncthing attempts to also synchronise filesystem extended
```
Comment (acolomb) [inline advanced/folder-sync-xattrs.rst:6]:
```suggestion
When enabled, Syncthing attempts to also synchronize filesystem extended
```

(not completely consistent in the docs, but the American spelling has more prior occurrences)

File: users/config.rst (hunk 3 above)
Comment (acolomb) [reply users/config.rst:313]: That is already fixed by #766, which also causes the conflicts right now.

**Review by acolomb — CHANGES_REQUESTED** [review]
The API additions for the `GET /rest/db/file` response should be added as well.  And the protocol (BEP) documentation seems largely out of sync, even more with the `PlatformData` added here.

**Comment by acolomb** [comment]
Do we have some kind of grand plan for the BEP and other protocol documentation? It seems very much out of sync with the protobuf specs and isn't coupled in any way.

**Comment by calmh** [comment]
Not beyond hopefully getting to updating it at some point.

**Pull request merged**